cmd/admin-handlers-config-kv.go

```diff
@@ -201,6 +207,18 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
+	// Check if callhome getting enabled, if so set the subnet proxy value to logger_webhook's subnet proxy
+	// Do this only if logger_wenhook's subnet proxy is not yet set
+	if result.SubSys == "callhome" && result.Cfg["callhome"]["_"].Get("enable") == "on" {
+		loggerWebhookSubnetProxy := result.Cfg["logger_webhook"]["subnet"].Get("proxy")
+		subnetProxy := result.Cfg["subnet"]["_"].Get("proxy")
+		if loggerWebhookSubnetProxy == "" && subnetProxy != "" {
```

Comment:
Might be better to check `loggerWebhookSubnetProxy != subnetProxy`. That way, even if some proxy was already set and it has been changed/removed on callhome, the same can be updated on the corresponding webhook.

Reply:
Ack